src/model/series.ts

```diff
@@ -351,7 +351,7 @@ export class Series<T extends SeriesType = SeriesType> extends PriceDataSource i
 		}
 
 		for (const customPriceLine of this._customPriceLines) {
-			res.push(customPriceLine.paneView());
+			res.concat(customPriceLine.paneViews());
```

Comment:
`concat` doesn't modify source array:

![image](https://user-images.githubusercontent.com/3112183/86798901-ff003180-c079-11ea-9fc9-cd0a27c8d951.png)

```suggestion
			res.push(...customPriceLine.paneViews());
```

Reply:
such a silly mistake, you are absolutely right :)